### PR TITLE
Update chainer.md

### PR DESCRIPTION
**Pull request by jspenc72**
- Removed Chainer Documentation Page
- Added link to Chainer Project
- Added note that Chainer is not supported in Kubeflow >0.6

Closes https://github.com/kubeflow/website/issues/1503

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1510)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
/hold

Hallo @jspenc72 Thank you for proposing this update. There are a few things we still need to discuss, as well as a few corrections needed in this PR, before we can make the change. I've applied a hold until the decisions are final. Let's continue the discussion on issue https://github.com/kubeflow/website/issues/1503

/cc @jlewi

File: content/docs/components/training/chainer.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 +++
 title = "Chainer Training"
-description = "See v0.7 docs for instrustions on using Chainer for training"
+description = "See v0.6 docs for instrustions on using Chainer for training"
```
Comment (sarahmaddox) [inline content/docs/components/training/chainer.md]: It's not clear what we mean by "v0.6 docs". We should specify that we mean **Kubeflow** v0.6 (as opposed to Chainer v0.6 or something else entirely). We should also include a link to the relevant v0.6 doc page. You can use the drop-down list on the Kubeflow docs to find v0.6, then navigate to the Chainer page to get the link for pasting into the docs.

Comment (sarahmaddox) [inline content/docs/components/training/chainer.md]: Fix typo "instrustions"

File: content/docs/components/training/chainer.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 weight = 4
 toc = true
 +++
 
-[Chainer](https://github.com/kubeflow/chainer-operator) is not supported in kubeflow 1.0
+[Chainer](https://github.com/kubeflow/chainer-operator) is not supported in kubeflow > 0.6
```
Comment (sarahmaddox) [inline content/docs/components/training/chainer.md]: Use a capital letter for **Kubeflow** as it's a product name. See the Kubeflow style guide: https://deploy-preview-1469--competent-brattain-de2d6d.netlify.com/docs/about/style-guide/#use-capital-letters-sparingly

Comment (sarahmaddox) [inline content/docs/components/training/chainer.md]: It's best not to use shorthand or abbreviations like `>` in technical docs. The abbreviations can be confusing and can cause errors in translation.

Change "> 0.6" to "versions greater than 0.6"

Also add a full stop at the end of the sentence.

File: content/docs/components/training/chainer.md (hunk 1 above)
Comment (jspenc72) [reply content/docs/components/training/chainer.md]: Done!

**Comment by jspenc72** [comment]
@sarahmaddox I have updated as per your request. https://github.com/kubeflow/website/compare/master...jspenc72:remove-chainer-docs?expand=1

**Pull request closed without merging**